shares.txt

```diff
@@ -1,57 +1,57 @@
 Add Share=Freigabe hinzufügen
 All files encrypted=Alle Dateien verschlüsselt
 All files protected=Alle Dateien geschützt
-All files unprotected=Dateien ungeschützt
+All files unprotected=Alle Dateien ungeschützt
 Allocation method=Verteilungsmethode
 Array must be Started to view Shares=Das Array muss **gestartet** sein, um die Freigaben zu sehen
-Automatically split any directory as required=Jedes Verzeichnis Splitten falls erforderlich
-Automatically split only the top five directory levels as required=Nur die obersten fünf Verzeichnisse Splitten falls erforderlich
-Automatically split only the top four directory levels as required=Nur die obersten vier Verzeichnisse Splitten falls erforderlich
-Automatically split only the top level directory as required=Nur das oberste Verzeichnis Splitten falls erforderlich
-Automatically split only the top three directory levels as required=Nur die obersten drei Verzeichnisse Splitten falls erforderlich
-Automatically split only the top two directory levels as required=Nur die obersten zwei Verzeichnisse Splitten falls erforderlich
-Calculated free space value=
+Automatically split any directory as required=Jedes Verzeichnis automatisch nach Bedarf aufteilen
+Automatically split only the top five directory levels as required=Nur die obersten fünf Verzeichnisse nach Bedarf aufteilen
+Automatically split only the top four directory levels as required=Nur die obersten vier Verzeichnisse nach Bedarf aufteilen
+Automatically split only the top level directory as required=Nur die oberste Verzeichnis-Ebene nach Bedarf aufteilen
+Automatically split only the top three directory levels as required=Nur die obersten drei Verzeichnisse nach Bedarf aufteilen
+Automatically split only the top two directory levels as required=Nur die obersten zwei Verzeichnisse nach Bedarf aufteilen
+Calculated free space value=Berechneter freier Speicherplatz
 Comments=Kommentare
-Compute All=Berechne alle
+Compute All=Alles berechnen
```

Comment:
Hier würde ich eher "Alle" in diesem Kontext benutzen, auf jeden Fall ist deine Übersetzung hier sinnvoller